src/main/scala/com/scalapenos/riak/internal/RiakHttpSupport.scala

```diff
@@ -17,9 +17,10 @@
 package com.scalapenos.riak
 package internal
 
-private[riak] trait RiakUriSupport {
-  import spray.http.Uri
-  import spray.http.Uri._
+private[riak] trait RiakHttpSupport {
+  import spray.http.{ Uri, HttpHeader, HttpHeaders, EntityTag }, HttpHeaders._, Uri._
```

Comment:
I'd prefer consistent wildcard imports instead of explicit imports but that's a personal thing and I'm not religious about it.